api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -360,6 +371,11 @@ message Bootstrap {
   // Envoy only supports ListenerManager for this field and Envoy Mobile
   // supports ApiListenerManager.
   core.v3.TypedExtensionConfig listener_manager = 37;
+
+  // Optional field to set the application logs format. If this field is set, it will override
+  // the default log format. In case the :option:`--log-format` command line option is used,
+  // it will override ``application_log_format`` configurations.
+  ApplicationLogFormat application_log_format = 38;
```

Comment:
The ``application_log_format`` would now support json format, but it can be extended to more types of formats by adding more options to the ``ApplicationLogFormat`` message - this is why I used ``oneof`` to allow extensibility of more types of formats